packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/ClearFilters/ClearFilters.js

```diff
@@ -0,0 +1,68 @@
+import { Box, Button } from 'grommet'
+import { Clear } from 'grommet-icons'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { SpacedText } from '@zooniverse/react-components'
+
+import counterpart from 'counterpart'
+import en from '../locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function Label () {
+  return (
+    <Box
+      align='center'
+      direction='row'
+      gap='xxsmall'
+    >
+      <Clear
+        size='small'
+      />
+      <SpacedText>
+        {counterpart('CharacteristicsFilter.clearFilters')}
+      </SpacedText>
+    </Box>
+  )
+}
+
+export default function ClearFilters (props) {
+  const {
+    handleFilter,
+    showingChoices,
+    totalChoices
+  } = props
+
+  return (
+    <Box
+      align='center'
+      direction='row'
+      fill='horizontal'
+      gap='xxsmall'
+      justify='center'
+      pad={{ top: 'xsmall' }}
+    >
+      <SpacedText>
+        {counterpart('CharacteristicsFilter.showing', { showing: showingChoices, total: totalChoices })}
+      </SpacedText>
+      <Button
+        disabled={showingChoices === totalChoices}
+        label={<Label />}
+        onClick={() => handleFilter()}
+        plain
+      />
```

Comment:
I'd suggest using `PlainButton` from the components library: https://zooniverse.github.io/front-end-monorepo/@zooniverse/react-components/index.html?path=/story/components-plainbutton--light-theme-default

It's very similar to how you've styled it, but has the hover and focus styling, using an underline, that is missing here.

Reply:
Refactored accordingly, thanks for suggestion!

Reply:
Sorry I wasn't clear, the `PlainButton` can be used instead of defining that `Label` component here too. I've added suggested change to clarify.